### ISSUE TITLE
Set a z-index.

### ISSUE THREAD

**Issue by ghost** [romgrk/termrk#79]
Hello

Would it be possible to add a z-index to the terminal? I noticed that tooltips and popovers displayed inside the editor tend to clip over the terminal, which seems to be fixed by setting a z-index explicitly (a value of 5 on the .termrk-panel class did the trick here).

Thanks in advance

**Comment by romgrk** [comment]
Something like this? https://github.com/romgrk/termrk/blob/branch-that-sets-a-zIndex-of-5-on-termrk-panel/styles/termrk.less#L22
I wouldn't know how to adjust, I no have that. But if it does the trick I'll include in the next patch.

**Comment by romgrk** [comment]
Ok, damn, no, not like that branch that doesnt do what it is supposed to do, but rather like this? https://github.com/romgrk/termrk/blob/branch-that-does-what-it-is-supposed-to-do/styles/termrk.less#L10

**Comment by ghost** [comment]
Yes, I think that should do the trick, performing the same change in the developer tools locally worked as intended.
